### PR TITLE
syncthing: Update to upstream release v1.19.0.

### PR DESCRIPTION
**Pull request by acolomb**
~~Not bumping the SPK revision as the current state has not been
published yet.~~

This was accidentally left out from #4930.

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Update-Policy#tests-checks)
- [ ] Any needed [documentation](https://github.com/SynoCommunity/spksrc/wiki/Create-documentation) is updated/created

### Type of change

- [ ] Bug fix
- [ ] New Package
- [x] Package update
- [ ] Includes small framework changes
- [ ] This change requires a documentation update (e.g. Wiki)

### CODE REVIEW

**Comment by acolomb** [comment]
I stand corrected.  Upstream version v1.19.0 was actually released as stable a few hours ago.  I had assumed it would be delayed.

File: spk/syncthing/Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ MAINTAINER = acolomb
 DESCRIPTION = Automatically sync files via secure, distributed technology.
 DESCRIPTION_FRE = Synchronisation automatique de fichiers via une technologie sécurisée et distribuée.
 DISPLAY_NAME = Syncthing
-CHANGELOG = "Update syncthing to v1.18.5.  Add wizard page to preconfigure credentials for the Web GUI.  Integrate with the certificate management UI in Control Panel."
+CHANGELOG = "Update syncthing to v1.18.6.  Add wizard page to preconfigure credentials for the Web GUI.  Integrate with the certificate management UI in Control Panel."
```
Comment (hgy59) [inline spk/syncthing/Makefile:15]:
```suggestion
CHANGELOG = "Update syncthing to v1.19.0.  Add wizard page to preconfigure credentials for the Web GUI.  Integrate with the certificate management UI in Control Panel."
```

**Comment by publicarray** [comment]
FYI I successfully used the internal updater on DSM7 after the newly published version. Is there really a need to bump the version again?

**Comment by acolomb** [comment]
It's not really a pressing need, but people updating now will get an SPK version that's two releases behind already.  Given it was just an unfortunate oversight, I wanted it fixed quickly.  Just to ease my mind :wink:

**Comment by hgy59** [comment]
> FYI I successfully used the internal updater on DSM7 after the newly published version. Is there really a need to bump the version again?

It's good to publish this as soon as possible to get an installer that asks for credentials and to avoid _downgrade_ for users of a previous version using the internal updater.

**Comment by acolomb** [comment]
Thanks, and again sorry for the mess from my side.

**Pull request merged**